test/test_GUNW.py

```diff
@@ -454,10 +453,13 @@ def test_hyp3_exits_succesfully_when_hrrr_not_available(mocker):
     # 2016-08-09 16:00:00 is a missing date
     mocker.patch('RAiDER.aria.prepFromGUNW.check_hrrr_dataset_availablity_for_s1_azimuth_time_interpolation',
                  side_effect=[False])
+    # The gunw id should not have a hyp3 file associated with it
+    # This call will still hit the HRRR s3 API as done in the previous test
+    mocker.patch("RAiDER.aws.get_s3_file", side_effect=['weirdly-named-prefix_-3ad24/S1-GUNW-A-R-106-tops-20160809_20140101-160001-00078W_00041N-PP-4be8-v3_0_0'])
```

Comment:
```suggestion
mocker.patch("RAiDER.aws.get_s3_file", side_effect=['hyp3-job-uuid-3ad24/S1-GUNW-A-R-106-tops-20160809_20140101-160001-00078W_00041N-PP-4be8-v3_0_0'])
```

Reply:
:thinking: shouldn't this patch also have a `.nc` on the end of it?

Reply:
Yes!